index.js

```diff
@@ -0,0 +1,10 @@
+var twitterText = require('twitter-text');
+
+module.exports = function(text, imageAttachments) {
+  var charsLeft = 140;
+
+  // Images (1 or more), use up 24 characters.
+  if (imageAttachments === true) { charsLeft -= 24; };
+
+  return charsLeft - twitterText.getTweetLength(text);
+};
```

Comment:
I updated my above code snippet with a few more suggestions (expanded if block to more lines for readability and removed extraneous semicolons). You could further remove all the semicolons in this snippet and it would continue to work as expected but I like semicolons for my own sanity (personal preference that is subject to change).